### PR TITLE
[MIG] base_3166: Migration to 11.0

### PR DESCRIPTION
**Pull request by etobella**
Migration of module. I also added a simple test.

### CODE REVIEW

**Review by yajo — APPROVED** [review]

File: base_iso3166/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import test_iso3166
```
Comment (pedrobaeza) [inline base_iso3166/tests/__init__.py]: Line at the end

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please check https://github.com/OCA/community-data-files/issues/22 about a change in the library API

**Comment by etobella** [comment]
@pedrobaeza, the requested change was done on 10.0 Migration: #25

**Comment by pedrobaeza** [comment]
Ah, OK, thanks

**Review by pedrobaeza — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Functional and technical review

**Comment by JordiBForgeFlow** [comment]
3 approvals. This one can be merged!

**Pull request merged**